content/en/v3/admin/platforms/k3s/_index.md

```diff
@@ -74,7 +96,7 @@ Note: If you get the error `Error enabling kubernetes auth: Post "https://127.0.
 - This bot user needs to have write permission to write to any git repository used by Jenkins X. This can be done by adding the bot user to the git organisation level or individual repositories as a collaborator Add the new bot user to your Git Organisation, for now give it Owner permissions, we will reduce this to member permissions soon.
 
 ### Jenkins X v3 installation
-
+- Make sure you have installed [jx 3.x binary](/v3/guides/jx3/) and put it on your `$PATH` as the `jx admin operator` will be used 
```

Comment:
Also add as pre-req, and link to the cli installation page: https://jenkins-x.io/v3/admin/setup/jx3/